envFiles/.env.production

```
@@ -23,10 +23,14 @@ API_JWT_EXPIRES_IN=900000
 API_REFRESH_TOKEN_EXPIRES_IN=604800000
 API_JWT_SECRET=
 API_COOKIE_SECRET=
-# REST auth (optional): JWT secret and TTLs in seconds for access/refresh tokens
+# REST auth: JWT secret (>= 32 chars) and TTLs in seconds; set API_AUTH_JWT_SECRET in production when using REST auth.
 # API_AUTH_JWT_SECRET=
 # API_ACCESS_TOKEN_TTL=900
 # API_REFRESH_TOKEN_TTL=2592000
+# Optional: cookie domain for cross-subdomain auth (e.g. .example.com).
+# API_COOKIE_DOMAIN=
+# Recommended for production (HTTPS): use secure cookies.
+API_IS_SECURE_COOKIES=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_IS_SECURE_COOKIES key should go before the API_JWT_EXPIRES_IN key

(UnorderedKey)